OctoAwesome/OctoAwesome/EntityList.cs

```diff
@@ -65,16 +66,12 @@ public IEnumerable<FailEntityChunkArgs> FailChunkEntity()
                 {
                     var position = entity.Components.GetComponent<PositionComponent>();
 
+                    Debug.Assert(position != null, nameof(position) + " != null");
                     if (position.Position.ChunkIndex.X != column.Index.X || position.Position.ChunkIndex.Y != column.Index.Y)
                     {
-                        yield return new FailEntityChunkArgs()
-                        {
-                            Entity = entity,
-                            CurrentChunk = column.Index,
-                            CurrentPlanet = column.Planet,
-                            TargetChunk = new Index2(position.Position.ChunkIndex),
-                            TargetPlanet = resourceManager.GetPlanet(position.Position.Planet),
-                        };
+                        yield return new FailEntityChunkArgs(entity: entity, currentChunk: column.Index,
```

Comment:
```suggestion
                        yield return new FailEntityChunkArgs(
                            entity: entity,
                            currentChunk: column.Index,
                            currentPlanet: column.Planet,
                            targetChunk: new Index2(position.Position.ChunkIndex),
                            targetPlanet: resourceManager.GetPlanet(position.Position.Planet));
```
Nur jeden 2. Parameter umzubrechen sieht komisch aus